python_modules/dagster/dagster/core/definitions/schedule_definition.py

```diff
@@ -33,6 +34,12 @@
     from .decorators.schedule import DecoratedScheduleFunction
 
 
+@whitelist_for_serdes
+class ScheduleStatus(Enum):
+    RUNNING = "RUNNING"
+    STOPPED = "STOPPED"
```

Comment:
Default prefix feels justified to avoid confusion with active state/status
`DefaultScheduleStatus`
`DefaultScheduleState`